visual_diff.py

```diff
@@ -138,10 +141,16 @@ def make_matrix(tokens_a, tokens_b):
         data_b = get_tokens(f_b.read(), language)
     matrix = make_matrix(data_a.tokens, data_b.tokens)
 
+    if args.color:
+        scores = find_duplicates.get_lengths(matrix, args.filename_b is None)
+    else:
+        scores = None
+
     if args.output_location is None:
```

Comment:
If the output location is set, we should probably still do something with colors. Think about that some more.